index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Náš tým</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body>
+
+
+    <h1>Náš tým</h1>
+
+    <div class="zeny">
+
+        <div class="hanka">

```

Comment:
Tady nemá smysl vytvářet třídu pro každou položku (holku) zvlášť 🙂
Vypadají všechny stejně a liší se jen obsahem? Pokud jo, stačí nám na to 1 třída. Žádná z holek se vizuálně neodlišuje, tak si pojďme ušetřit psaní - to je právě skvělá vlastnost tříd, že je můžeme používat opakovaně 🙂